<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>995</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>995</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="cam1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Camara 1</string>
    </property>
    <widget class="QTextEdit" name="C1_show">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>950</width>
       <height>150</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="cam1_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>971</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Camara 2</string>
    </property>
    <widget class="QTextEdit" name="C2_show">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>950</width>
       <height>150</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="linea_IP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>810</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>810</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="cam1_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>971</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>MQTT (publicaciones)</string>
    </property>
    <widget class="QTextEdit" name="mqtt_show">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>950</width>
       <height>150</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="cam1_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>971</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>MQTT (recibiendo mensajes)</string>
    </property>
    <widget class="QTextEdit" name="py_show">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>950</width>
       <height>150</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1190</x>
     <y>991</y>
    </hint>
    <hint type="destinationlabel">
     <x>1116</x>
     <y>991</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
